SS_benchfore.tpl

```diff
@@ -1060,7 +1062,9 @@ FUNCTION void Get_Benchmarks(const int show_MSY)
 
     Btgt_Fmult = Fmult;
     if (rundetail > 0 && mceval_counter == 0 && show_MSY == 1)
-      cout << " got_Btgt " << Btgt_Fmult << " " << Btgt / SSB_unf << endl;
+    {
+      echoinput << "Finished Btgt: " << Btgt_Fmult << " " << Btgt / SSB_unf << endl;
```

Comment:
Is there a difference between "Finished" and "Calculated"? If no, then I suggest changing "Finished" to "Calculated".

Reply:
correct.  Please change it to Calculated